common/cpp/crypto/verify_signature.cpp

```diff
@@ -18,16 +18,17 @@
 #include <string.h>
 #include <openssl/x509v3.h>
 #include <openssl/pem.h>
+#include <openssl/evp.h>
```

Comment:
This was not included in crypto_utils.cpp before. Is inclusion of this header file needed?